profiles/2020-09-01-hybrid/azure/src/main/java/com/microsoft/azure/management/profile_2019_03_01_hybrid/CustomCipherSuites.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.profile_2019_03_01_hybrid;
+
+import okhttp3.CipherSuite;
+import org.apache.commons.lang3.ArrayUtils;
+
+/**
+ * Collects cipher suites in OkHttp for needed.
+ */
+public class CustomCipherSuites {
```

Comment:
Just curious, any background on this?

Reply:
Yes, I am sending email for them.

For Azure Stack, the default `CipherSuites` in `OkHttp` cannot send successful request. (Due to their security concern)